Sample Solutions/Viva Insights Query Analysis Automation/Notebooks/vivaInsights_meeting.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "source": [
+        "# # #Initializing the parameters\r\n",
+        "# # Read about how a Parameter cell should be used for definign and initializing\r\n",
+        "# # parameters in Synapse\r\n",
+        "\r\n",
+        "# Storage Account Name\r\n",
+        "StorageAccountName = \"\"\r\n",
+        "# Main container/directory on the storage account\r\n",
+        "VivaInsightsDataFileSystem = \"\"\r\n",
+        "\r\n",
+        "PipelineId = \"\"\r\n",
+        "MeetingQueryDatasetFolder = \"\"\r\n",
+        "SecondaryEmployeeId = \"\"\r\n",
+        "\r\n",
+        "# Database connection information\r\n",
+        "SQLServerEndpoint = \"\"\r\n",
+        "DBName = \"\"\r\n",
+        "DBUser = \"\"\r\n",
+        "DBPass = \"\"\r\n",
+        "DBPort = \"\"\r\n",
+        "\r\n",
+        "\r\n"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "source_hidden": false,
+          "outputs_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "tags": [
+          "parameters"
+        ]
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import sys\r\n",
+        "import json\r\n",
+        "\r\n",
+        "\r\n",
+        "from pyspark import SparkContext, SparkConf\r\n",
+        "from pyspark.sql import SQLContext\r\n",
+        "from pyspark.sql.functions import explode\r\n",
+        "from pyspark.sql.functions import *\r\n",
```

Comment:
think import * would cover all the names. it could also cause some name collision if there were any other modules with the same name function